package.json

```diff
@@ -111,6 +111,7 @@
     "@jest/test-sequencer": "^28.1.1",
     "@types/babel__core": "^7.1.15",
     "@types/babel__preset-env": "^7.9.2",
+    "@types/bitcoinjs-lib": "^5.0.0",
```

Comment:
please remove, bitcoinjs-lib provides its own type definitions

Reply:
actually that works now. weird that may have been an IDE issue